tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -148,29 +148,29 @@ class TestExpectTableRowCountToEqualOtherTable:
         data_source_configs=[
             PostgreSQLDatasourceTestConfig(
                 column_types={"col_a": sqltypes.INTEGER},
-                extra_assets={"test_table_two": {"col_b": sqltypes.VARCHAR}},
+                extra_assets={"test_table_a": {"col_b": sqltypes.VARCHAR}},
```

Comment:
TODO in a subsequent task: figure out how to generate these names ourselves like we do for the main asset.